modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +189,33 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+A Mule application can contain several batch job definitions. Each definition 
+has its own scheduling strategy. A *Scheduling Strategy* enables you to control 
+how instances of a given batch job are executed: 
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
```

Comment:
GLOBALLY convert all lists of this type, going forward, to appear as follows:
```suggestion
* ORDERED_SQUENTIAL (the default)
If several job instances are in an executable
```

Reply:
doing that requires a `+` after the bullet line to keep the formatting correct.